client/my-sites/plans-features-main/index.tsx

```diff
@@ -651,6 +657,7 @@ const PlansFeaturesMain = ( {
 			<QuerySites siteId={ siteId } />
 			<QuerySitePlans siteId={ siteId } />
 			<QueryActivePromotions />
+			<QueryProductsList />
```

Comment:
**Note:**

`<QueryProductList />` is necessary to generate add-on product data on the `/plans` page